backend/integration.go

```diff
@@ -2756,3 +2758,74 @@ func testStaffInvasion(s *serverUnderTest) {
 			`{"session_id":"%s","id":"*","name":"","server_id":"*","server_era":"*","is_staff":true,"is_manager":true}`, c2.sessionID)
 	})
 }
+
+func testNotifyUser(s *serverUnderTest) {
+	Convey("Successful login disconnects all sessions associated with user", func() {
+		ctx := scope.New()
+		kms := s.app.kms
+
+		// Create manager account and room.
+		nonce := fmt.Sprintf("notify-%s", time.Now())
+		cammie, _, err := s.Account(ctx, kms, "email", "cammie"+nonce, "cammiepass")
+		So(err, ShouldBeNil)
+
+		// Create an initial connection
+		conn1 := s.Connect("notify1")
+		defer conn1.Close()
+		conn1.expectPing()
+		conn1.expectSnapshot(s.backend.Version(), nil, nil)
+
+		// Create a second connection with the same cookie
+		conn2 := conn1
```

Comment:
This should be `conn2 := *conn1` (or something equivalent). Because conn2 is a pointer to the same address as conn1, the call to s.Reconnect on the next line replaces conn1's websocket.